.github/workflows/ci-build-proto.yml

```diff
@@ -0,0 +1,32 @@
+name: "build protobufs"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
This is more of a question than a comment: Does this mean this will be run on any PR people open, even if the PR is coming from some random fork? But only if the PR is against the master branch?

Reply:
Correct.